lms/static/scripts/frontend_apps/components/dashboard/test/LastSyncIndicator-test.js

```diff
@@ -0,0 +1,39 @@
+import { FileCodeIcon, formatDateTime } from '@hypothesis/frontend-shared';
+import { mount } from 'enzyme';
+
+import LastSyncIndicator from '../LastSyncIndicator';
+
+describe('LastSyncIndicator', () => {
+  function createComponent(dateTime) {
+    return mount(
+      <LastSyncIndicator
+        icon={FileCodeIcon}
+        taskName="task"
+        dateTime={dateTime}
+      />,
+    ).find('[data-testid="container"]');
+  }
+
+  [
+    { dateTime: null, expectedTitle: undefined },
+    {
+      dateTime: '2024-10-02T14:24:15.677924+00:00',
+      expectedTitle: `task last synced on ${formatDateTime('2024-10-02T14:24:15.677924+00:00', { includeWeekday: true })}`,
+    },
+  ].forEach(({ dateTime, expectedTitle }) => {
+    it('has title when dateTime is not null', () => {
+      const wrapper = createComponent(dateTime);
+      assert.equal(wrapper.prop('title'), expectedTitle);
+    });
+
+    it('shows syncing when date time is null', () => {
```

Comment:
```suggestion
    it('shows syncing message when date time is null', () => {
```

Make it clearer that "syncing" refers to a piece of UI text.